### ISSUE TITLE
Compiling from source : Windows | File not found when exists.

### ISSUE THREAD

**Issue by meharshjoshi93** [RobotLocomotion/drake-distro#19]
The issue appears to be happening consistently on my machine : Windows 10 x64 | MATLAB 2014a x64 | Visual Studio 2015 Community | Windows 10 SDK | Windows 7.1 SDK |
executed as : make -j5 ( even with 'make' and 'make -j4')

The error is : 
![error](https://cloud.githubusercontent.com/assets/6691677/11109946/6274a862-891c-11e5-9a27-567135fbfe03.jpg)

However the file exists (so do the other 40 some files!) : 
![exists](https://cloud.githubusercontent.com/assets/6691677/11109948/6513fbf4-891c-11e5-8bab-7bf03b8abc23.jpg)

Some research led me to this SO link : 
http://stackoverflow.com/questions/21408415/main-cc530-fatal-error-folder-file-h-no-such-file-or-directory

I made sure all the install  prerequisites as mentioned in : 
https://github.com/RobotLocomotion/drake/wiki/Windows 
are in order. And mex is configured to use Microsoft C++ Compiler from the SDK.

tried running cmd as administrator and default both!

**Comment by RussTedrake** [comment]
Does the error persist when you run make a second time?  Those files are generated by the build, and if they still have a lock on them could cause this error.  But then you would expect run make a second time to succeed.

**Comment by meharshjoshi93** [comment]
Yes the error still persists, albeit with a modified error message:

![warning_reduced_errors_persist](https://cloud.githubusercontent.com/assets/6691677/11130620/01b51c92-89ac-11e5-9bbd-a8b70d4abbde.jpg)

The number of warnings have reduced however the number of errors are still the same.
